README.md

```diff
@@ -2,5 +2,247 @@
 
 [![Travis Build Status][build-badge]][build]
 
+This is a library of some custom validators for React components properties.
+Initially they were part of [React-Bootstrap](https://github.com/react-bootstrap/react-bootstrap/) project.
+
+### Usage
+
+All validators can be imported as
+```js
+import elementType from 'react-prop-types/elementType';
+// or
+import { elementType } from 'react-prop-types';
+...
+propTypes: {
+  someProp: elementType
+```
+or
+```js
+import CustomPropTypes from 'react-prop-types';
+// and then used as usual
+propTypes: {
+  someProp: CustomPropTypes.elementType
+```
+
+If you use `webpack`, you can bundle only those validators you need like this:
+```js
+import elementType from 'react-prop-types/elementType'
+```
+
+---
+#### all(arrayOfValidators)
+
+This validator allows to have complex validation like this:
+```js
+propTypes: {
```

Comment:
Perhaps the example can be simplified a bit and become react-bootstrap agnostic? That might be stuff for a later PR however.